consecutive ones/consecutive_ones.cpp

```diff
@@ -0,0 +1,56 @@
+#include <iostream> 
+#include  <vector> 
+#include  <climits>  
+
+
```

Comment:
Would be great if you can explain your approach in brief using comments

Reply:
okay willbe doing  in next questions coming  up